rustls/src/builder.rs

```diff
@@ -104,16 +104,21 @@ pub struct ConfigBuilder<Side: ConfigSide, State> {
     pub(crate) side: PhantomData<Side>,
 }
 
+fn friendly_names(raw_type: &str) -> (&str, &str) {
```

Comment:
Style nit: should go below the `Debug` impl, instead of above it. Maybe even near the bottom of the module, if there will soon be other `Debug` impls in this module that need it?

Reply:
:crossed_fingers: I'm hoping I can use your idea of adding an associated type to `ConfigSide` to remove the need for this change but in the meantime I've moved it to the bottom of the module.